docs/fundamentals/code-analysis/quality-rules/ca1303.md

```diff
@@ -48,9 +48,15 @@ To fix a violation of this rule, replace the string literal with a string retrie
 
 ## When to suppress warnings
 
-It is safe to suppress a warning from this rule if the code library will not be localized or if the string is not exposed to the end user or a developer using the code library.
+It's safe to suppress a warning from this rule if either of the following statements applies:
 
-Users can eliminate noise against methods that should not be passed localized strings by either renaming the parameter or property, or by marking these items as conditional.
+- The code library will not be localized.
+- The string is not exposed to the end user or a developer using the code library.
+
+You can eliminate noise against methods that shouldn't be passed localized strings in one of the following ways:
```

Comment:
```suggestion
## How to suppress warnings

You can eliminate unnecessary warnings for methods that don't require localized strings in the following ways:
```

Reply:
I used the wording you suggested for the intro, but wanted to change the heading since it's not really suppressing warnings, it's actually "fixing" the code so they don't generate warnings.